src/components/LoanCards/AlgoliaLoanCardAdapter.vue

```diff
@@ -143,9 +143,3 @@ export default {
 	}
 };
 </script>
-
-<style lang="scss">
```

Comment:
Removing this probably affects the grid loan card display on /algolia-vue but seeing as that's all going to get a re-work soon it shouldn't be an issue.